Entities/MOSRotating.cpp

```diff
@@ -1494,79 +1423,50 @@ void MOSRotating::PostTravel()
 //////////////////////////////////////////////////////////////////////////////////////////
 // Description:     Updates this MOSRotating. Supposed to be done every frame.
 
-void MOSRotating::Update()
-{
-
-#if defined DEBUG_BUILD || defined MIN_DEBUG_BUILD
+void MOSRotating::Update() {
 	RTEAssert(m_MOID == g_NoMOID || (m_MOID >= 0 && m_MOID < g_MovableMan.GetMOIDCount()), "MOID out of bounds!");
-#endif
 
     MOSprite::Update();
 
-	if (m_InheritEffectRotAngle)
-		m_EffectRotAngle = m_Rotation.GetRadAngle();
-
-    // Align the orienatation with the velocity vector, if any
-    if (m_OrientToVel > 0 && m_Vel.GetLargest() > 5.0)
-    {
-        // Clamp
-        if (m_OrientToVel > 1.0)
-            m_OrientToVel = 1.0;
+    if (m_InheritEffectRotAngle) {
+        m_EffectRotAngle = m_Rotation.GetRadAngle();
+    }
 
-        // Velocity influence
-        float velInfluence = m_OrientToVel < 1.0 ? m_Vel.GetMagnitude() / 100 : 1.0f;
-        if (velInfluence > 1.0)
-            velInfluence = 1.0;
+    if (m_OrientToVel > 0 && m_Vel.GetLargest() > 5.0) {
+        m_OrientToVel = std::clamp(m_OrientToVel, 0.0F, 1.0F);
 
-        // Figure the difference in current velocity vector and 
+        float velInfluence = std::clamp(m_OrientToVel < 1.0 ? m_Vel.GetMagnitude() / 100.0F : 1.0F, 0.0F, 1.0F);
```

Comment:
As I recall, parentheses were a take it or leave it one - i.e. you wanted it and I didn't (except in the potential case of really complex conditions) and we agreed to split the difference.

Reply:
You trying to start a civil war?

Reply:
Hahaha from the start this was an either way goes cause I didn't like bracketing every tiny ternary. As I recall, I said "I'm not doing it but if you wanna come back afterwards and do it yourself, I won't stop you".
So yeah, civil war time I guess lol. We can split CCCP into two discord servers instead of one, get some real drama going :P

Reply:
Bich please. He who controls the emotes controls the CCCP.
Your server will be DOA while my server will prosper.

We agreed that if it's `a ? b : c` then no parentheses and if `a = b ? c : d` then `(a = b) ? c : d`.

Reply:
I'd put `a == b ? c : d` in the same complexity level as `a ? b : c`
If it's something like `(a == b || a == f || (a ==e && a == g)) ? b : c` I'm 100% in favour of brackets cause you need em, but I find with simple conditions like this they make it less readable for me.
Anyway, yeah, I'mma leave this to you like we agreed way back when. If you wanna go back and change it I won't stop you, but I'm not doing it cause if I do it once you'll be telling me to do it every time and I don't wanna :D